contrib/python/src/python/pants/contrib/python/checks/checker/checker.py

```diff
@@ -41,7 +41,7 @@ def noqa_file_filter(python_file):
   return any(_NOQA_FILE_SEARCH(line) is not None for line in python_file.lines)
 
 
-class Checker(object):
+class Checker:
```

Comment:
This entire module must keep Python 2 compatibility because it is ran as a pex that may be invoked with Python 2. The BUILD file explains this, but I haven't thought of a better way to enforce it :/ See https://github.com/pantsbuild/pants/issues/7979 for trying to start testing this with Py2 so that we can catch any bad changes like this.

Reply:
@Eric-Arellano this synatax is perfectly valid in python 2.7:

<img width="612" alt="Screenshot 2019-08-16 18 57 42" src="https://user-images.githubusercontent.com/1268088/63205331-bd2c5e00-c057-11e9-9b3a-e0d3edb0394b.png">

Reply:
Yes, but not desirable. In Py2, if you don't extend `object` then it is an old-style class: https://stackoverflow.com/questions/54867/what-is-the-difference-between-old-style-and-new-style-classes-in-python

Reply:
Fair point. I will update this.